flink-table/flink-table-planner/src/main/java/org/apache/flink/table/planner/alias/ClearJoinHintsWithInvalidPropagationShuttle.java

```diff
@@ -48,19 +49,10 @@
  * <p>TODO some node will be attached join hints when parse SqlNode to RelNode such as Project and
  * etc. The join hints on these node can also be cleared.
  */
-public class ClearJoinHintWithInvalidPropagationShuttle extends RelShuttleImpl {
+public class ClearJoinHintsWithInvalidPropagationShuttle extends JoinHintsRelShuttle {
```

Comment:
Nit: Do you think it's better to move this class under package`org.apache.flink.table.planner.hint`?

Reply:
That sounds better.